great_expectations_cloud/agent/agent.py

```diff
@@ -114,7 +114,7 @@ def run(self) -> None:
 
         print("Opening connection to GX Cloud.")
         self._listen()
-        print("Connection to GX Cloud has been closed.")
+        print("The connection to GX Cloud has been closed.")
```

Comment:
Updated some of the comments in this file to force CI to bump the version and test if the above change works :)

Reply:
It worked! https://github.com/great-expectations/cloud/actions/runs/8850366544/job/24304509999?pr=233